### PR TITLE
Upgrade to use Quarkus 1.13.4.Final

### PR DESCRIPTION
**Pull request by Sgitario**
- Added workaround for https://github.com/quarkusio/quarkus/issues/17226

### CODE REVIEW

**Comment by Sgitario** [comment]
run tests

**Comment by Sgitario** [comment]
All the tests worked fine but "Quarkus OpenShift TS: SQL Database: Multiple Persistence Units" which I tried again locally in Native and it indeed worked.

**Comment by jsmrcka** [comment]
> All the tests worked fine but "Quarkus OpenShift TS: SQL Database: Multiple Persistence Units" which I tried again locally in Native and it indeed worked.

Interesting, I did not run into this. Locally, everything works for me except for testing against quarkus main, which is irrelevant to this PR.

File: security/keycloak-authz/src/main/resources/application.properties [hunk 1]
```diff
@@ -9,6 +9,9 @@ quarkus.oidc.token.lifespan-grace=60
 quarkus.keycloak.policy-enforcer.enable=true
 quarkus.keycloak.policy-enforcer.paths.health.path=/q/*
 quarkus.keycloak.policy-enforcer.paths.health.enforcement-mode=DISABLED
+// TODO: Workaround for https://github.com/quarkusio/quarkus/issues/17179
```
Comment (jsmrcka) [inline security/keycloak-authz/src/main/resources/application.properties:12]: Just curious: is this a legal comment syntax? I mean, it obvoiusly works, but still...

**Pull request merged**